### PR TITLE
Release_0.1.0 -> main

### PR DESCRIPTION
**Pull request by varsha888**
Merge release v0.1.0 to main

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,4 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-[Unreleased]: https://github.com/mobilecoinfoundation/build-rs/compare/v0.0.0...HEAD
+[Unreleased]: https://github.com/mobilecoinfoundation/sgx-sigstruct/compare/v0.1.0...HEAD
+
+## [0.1.0] - 2023-04-27
+
+### Added
+
+- Add Cargo build script helpers.
+
+[0.1.0]: https://github.com/mobilecoinfoundation/build-rs/compare/v0.0.0...v0.1.0
```
Comment (nick-mobilecoin) [inline CHANGELOG.md]: :wrench: Looks like this got a bit mangled

```suggestion
## [0.1.0] - 2023-04-27

### Added

- Add Cargo build script helpers.

<!-- next-url -->
[Unreleased]: https://github.com/mobilecoinfoundation/sgx-sigstruct/compare/{{tag_name}}...HEAD
[0.1.0]: https://github.com/mobilecoinfoundation/build-rs/compare/v0.1.0
```

**Review by nick-mobilecoin — APPROVED** [review]
Some minor changelog fixes

Note: This repo is currently set to not allow merge commits, yet the tag on this branchline will persist. 
We can let releases sit out on their own, or we can temporarily allow merge commits

**Comment by varsha888** [comment]
> Some minor changelog fixes
> 
> Note: This repo is currently set to not allow merge commits, yet the tag on this branchline will persist. We can let releases sit out on their own, or we can temporarily allow merge commits

Got it, I've changed to allow merge commits

**Comment by nick-mobilecoin** [comment]
> > Some minor changelog fixes
> > Note: This repo is currently set to not allow merge commits, yet the tag on this branchline will persist. We can let releases sit out on their own, or we can temporarily allow merge commits
> 
> Got it, I've changed to allow merge commits

It looks like there was a force push, that's going to leave the tag out on its own (which is fine), but if we're going to do that then going back to merge squash is probably the way to go
image

**Review by awygle — APPROVED** [review]

**Pull request merged**